README.md

```diff
@@ -11,7 +11,7 @@ This library is an unofficial plugin for the [Aurelia](http://www.aurelia.io/) p
 ## Overview
 
 Sick of configuring every plugin separately, inconsistent naming and one plugin not knowing what the other does. Here comes the solution. One app config file for all your plugin. It works as follows:
-At aurelia configuration you tell aurelia-config which plugin defaults it should load and the app config objects you want to merge into that sequentially. Then it (optionally) calls sequentially the plugin configurations on them with the (plugin namepsaced) merged settings. 
+At aurelia configuration you tell aurelia-config which plugin config it should load and the app config objects you want to merge into that sequentially. Then it (optionally) calls sequentially the plugin configurations on them with the (plugin namepsaced) merged settings. 
```

Comment:
Slightly change the readme. I've written a tiny part to show you what I mean: https://gist.github.com/RWOverdijk/b3a5e875e65d70a05a9788c8774b2e05